Pear/View/EndUserLicenseAgreementView.swift

```diff
@@ -0,0 +1,129 @@
+                    NSRange.init(location: 0, length: licenseButtonText.length))
+        licenseButton.setAttributedTitle(licenseButtonText, for: .normal)
+        licenseButton.setTitleColor(.darkGreen, for: .normal)
+        licenseButton.titleLabel?.font = ._16CircularStdBook
+        licenseButton.addTarget(self, action: #selector(licenseButtonPressed), for: .touchUpInside)
+        addSubview(licenseButton)
+
+        acceptButton.setTitle("Accept", for: .normal)
+        acceptButton.setTitleColor(.white, for: .normal)
+        acceptButton.titleLabel?.font = ._20CircularStdBold
+        acceptButton.isEnabled = true
+        acceptButton.layer.cornerRadius = Constants.Onboarding.mainButtonSize.height / 2
+        acceptButton.addTarget(self, action: #selector(acceptButtonPressed), for: .touchUpInside)
+        addSubview(acceptButton)
+        
+        pearImageView.image = UIImage(named: "heartPear")
+        pearImageView.contentMode = .scaleAspectFill
+        pearImageView.layer.shadowColor = UIColor.black.cgColor
+        pearImageView.layer.shadowOffset = CGSize(width: 0.0, height: 3.0)
+        pearImageView.layer.shadowOpacity = 0.15
+        pearImageView.layer.shadowRadius = 7.25
+        addSubview(pearImageView)
+    }
+    
+    private func setupConstraints() {
+        let verticalPadding = 20
+        let textIteritemSpacing = 5
+        let acceptButtonSize = CGSize(width: 280, height: 55)
+        let licenseButtonSize = CGSize(width: 250, height: 20)
+        let pearImageSize = CGSize(width: 115, height: 118)
+
+        pearImageView.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.size.equalTo(pearImageSize)
+            make.top.equalToSuperview().offset(verticalPadding + textIteritemSpacing)
+        }
+        
+        headerLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.centerY.equalToSuperview()
+        }
+        
+        descriptionLabel.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalTo(headerLabel.snp.bottom).offset(3 * textIteritemSpacing)
+        }
+        
+        licenseButton.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.size.equalTo(licenseButtonSize)
+            make.top.equalTo(descriptionLabel.snp.bottom).offset(textIteritemSpacing)
+        }
+        
+        acceptButton.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.size.equalTo(acceptButtonSize)
+            make.bottom.equalToSuperview().inset(verticalPadding)
+        }
+        
+    }
+    
+    @objc private func licenseButtonPressed() {
+        guard let url = URL(string: licenseURL) else { return }
+        delegate?.presentUserLicenseAgreementWebView(url: url)
+    }
+    
+    @objc private func acceptButtonPressed() {
+        UserDefaults.standard.set(true, forKey: "hasAcceptedTerms")
```

Comment:
Might also want to put all of our userdefault keys into a constants file, but this is something you can hand off to Tiffany is more quality of life